scalableminds/webknossos
# Add shortcut hints to context menu

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Open a hybrid explorative tracing.
- Test every shortcut given by the context menu, whether the shortcuts are correct.

- After that please go through all context menu items and check whether I shortcut hints for every context option that has a shortcut. [Here](https://docs.webknossos.org/reference/keyboard_shortcuts) is the shortcut doc.

### Issues:
- fixes #5619

------
